lib/PlacesProvider.js

```diff
@@ -430,6 +447,89 @@ Links.prototype = {
     }
     return null;
   }),
+
+  /**
+   * Obtain a set of links for highlights
+   *
+   * @param {Object} options
+   *        {Integer} limit: Maximum number of results to return. Max 20
+   *
+   * @returns {Promise} Returns a promise with the array of links as payload.
+   */
+  getHighlightsLinks: Task.async(function*(options = {}) {
+
+    let {limit} = options;
+    if (!limit || limit.options > HISTORY_RESULTS_LIMIT) {
+      limit = HISTORY_RESULTS_LIMIT;
+    }
+
+    let params = {limitBookmarks: 1, limitHistory: (limit - 1)};
+
+    let sqlQuery = `SELECT DISTINCT * FROM (
+                      SELECT * FROM (
+                        SELECT p.url as url,
+                               f.data as favicon,
+                               f.mime_type as mimeType,
+                               p.title as title,
+                               p.frecency as frecency,
+                               p.last_visit_date / 1000 as lastVisitDate,
+                               "bookmark" as type,
+                               b.id as bookmarkId,
+                               b.guid as bookmarkGuid,
+                               b.title as bookmarkTitle,
+                               b.lastModified / 1000 as lastModified,
+                               b.dateAdded / 1000 as bookmarkDateCreated
+                        FROM moz_places p
+                        INNER JOIN moz_bookmarks b
+                          ON b.fk = p.id
+                        LEFT JOIN moz_favicons f
+                          ON p.favicon_id = f.id
+                        WHERE date(b.dateAdded / 1000 / 1000, 'unixepoch') > date('now', '${HIGHLIGHTS_THRESHOLDS.created}')
```

Comment:
Yes, the goal is to obtain a bookmark that has been created within the last 3 days